service/controller/resource/monitoring/remotewritesecret/resource.go

```diff
@@ -0,0 +1,81 @@
+package remotewritesecret
```

Comment:
I'm a bit confused 😕 
Why this package? `remotewritesecret` while the logic is added in `remotewriteagentsecret`

Reply:
I think this one needs to go 😓 
Too many refactoring between the first and the last iteration